### PR TITLE
Added ElMomentoDelMogus command

### PR DESCRIPTION
**Pull request by NilEis**

### CODE REVIEW

File: src/main/java/ml/codeboy/thebot/CommandHandler.java [hunk 1]
```diff
@@ -219,6 +219,7 @@ public void registerKnowCommands() {
         registerCommand(new ShittyTranslateCommand());
         registerCommand(new ASCIICommand());
         registerCommand(new GifCommand());
+        registerCommand(new ElMomentoCommand());
```
Comment (the-codeboy) [inline src/main/java/ml/codeboy/thebot/CommandHandler.java:222]:
```suggestion
```

File: src/main/java/ml/codeboy/thebot/commands/ElMomentoCommand.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    public ElMomentoCommand() {
+        super("ElMomentoDelMogus", "El Momento del mogus", "elmomento", "mogus");
+    }
+
+    @Override
+    public void run(CommandEvent event) {
+        event.reply("https://tenor.com/view/el-momento-del-mogus-amog-mogus-el-momento-gif-22890012");
+    }
+}
```
Comment (the-codeboy) [inline src/main/java/ml/codeboy/thebot/commands/ElMomentoCommand.java:17]:
```suggestion
```

**Review by the-codeboy — CHANGES_REQUESTED** [review]
I just need a few changes and I will merge it

**Comment by the-codeboy** [comment]
If you don't respond I will close this PR as inactive

**Comment by NilEis** [comment]
I do not believe that your request is feasible.

**Comment by the-codeboy** [comment]
In that case you may make a better suggestion if it suits you dear sir

File: src/main/java/ml/codeboy/thebot/commands/ElMomentoCommand.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package ml.codeboy.thebot.commands;
+
+import ml.codeboy.thebot.events.CommandEvent;
+import ml.codeboy.thebot.events.SlashCommandCommandEvent;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+
+public class ElMomentoCommand extends Command {
```
Comment (the-codeboy) [inline src/main/java/ml/codeboy/thebot/commands/ElMomentoCommand.java]:
```suggestion
public class ElMomentoCommand extends DebugCommand {
```

**Review by the-codeboy — APPROVED** [review]
Looks good

**Pull request merged**